.github/workflows/lint.yml

```diff
@@ -10,13 +10,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v1
-    - name: set PY
```

Comment:
The pre-commit GHA now includes caching built in :+1: 

https://github.com/pre-commit/action/pull/31